client/cpu9p_windows.go

```diff
@@ -0,0 +1,141 @@
+// Copyright 2018 The gVisor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
You should mention Apache in the top-most LICENSE file -- some people might be surprised that some files are under different licenses.